renku/cli/utils/click.py

```diff
@@ -31,3 +31,37 @@ def convert(self, value, param, ctx):
         if value is None:
             return None
         return super(CaseInsensitiveChoice, self).convert(value.lower(), param, ctx)
+
+
+class MutuallyExclusiveOption(click.Option):
+    """Custom option class to allow specifying mutually exclusive options in click commands."""
+
+    def __init__(self, *args, **kwargs):
+        mutually_exclusive = sorted(kwargs.pop("mutually_exclusive", []))
+        self.mutually_exclusive = set()
+        self.mutually_exclusive_names = []
+
+        for mutex in mutually_exclusive:
+            if type(mutex) == tuple:
+                self.mutually_exclusive.add(mutex[0])
+                self.mutually_exclusive_names.append(mutex[1])
+            else:
+
+                self.mutually_exclusive.add(mutex)
+                self.mutually_exclusive_names.append(mutex)
+
+        help = kwargs.get("help", "")
```

Comment:
Minor thing, but I believe we should stay away from shadowing built-in names. Maybe rename this to `_help`?